main.py

```
@@ -21,5 +21,4 @@ def read_root():
 @app.post("/predict")
 def predict(data: DiabetesInput):
     input_data = np.array([[data.Pregnancies, data.Glucose, data.BloodPressure, data.BMI, data.Age]])
-    prediction = model.predict(input_data)[0]
-    return {"diabetic": bool(prediction)}
+    prediction = model.predict(input_data)[0]
```

Ruff (0.11.9)

Local variable `prediction` is assigned to but never used

Remove assignment to unused variable `prediction`

(F841)